src/soroban/api.ts

```diff
@@ -175,8 +175,21 @@ export namespace Api {
     value: string;
   }
 
-  export interface RequestAirdropResponse {
-    transaction_id: string;
+  interface RawLedgerEntryChange {
+    type: number;
+    // This is LedgerKey in base64
+    key: string;
+    // these are xdr.LedgerEntry in base64
+    before?: string;
+    // these are xdr.LedgerEntry in base64
+    after?: string;
```

Comment:
these are always present but might be `null` (which is distinct from `undefined`)